setup.rst

```diff
@@ -62,6 +62,28 @@ to meet those requirements.
     If the installer doesn't work for you or doesn't output anything, make sure
     that the PHP `Phar extension`_ is installed and enabled on your computer.
 
+.. note::
+
+    If the SSL certificates are not properly installed in your system, you
+    may get this error:
+
+        cURL error 60: SSL certificate problem: unable to get local issuer certificate.
+
+    You can solve this issue as follows:
+
+    #. Download a file with the updated list of certificates from
+       https://curl.haxx.se/ca/cacert.pem
```

Comment:
As far as I understand this will only work as long as the URL detection doesn't break, won't it? Do we rely on this anywhere else in the docs?